src/main/java/eggventory/commands/help/HelpCommand.java

```diff
@@ -48,18 +50,32 @@ public String execute(StockList list, Ui ui, Storage storage) {
         String output = "";
         String filename;
         if (this.options == null) {
-            filename = "\\src\\main\\java\\eggventory\\commands\\help\\Help.txt";
+            if(OS.indexOf("win") >= 0){

```

Comment:
should be able to handle this within an external method something like this
String filename =getFilePath(os) + "Help.txt"; 

then can reuse for each file also

Reply:
Oh cool, thanks for this

Reply:
Resolved, we used getResource.class